multipage.js

```diff
@@ -116,8 +116,7 @@ for(var i=0; i<sections.length; i++) {
 
   // find the proper section
   section.node = header.parent();
-  //while (section.node.get(0).tagName !== "section") {
-  while (section.node.get(0) && section.node.get(0).tagName !== "section") {
+  while (typeof section.node[0] !== 'undefined' && section.node.get(0).tagName !== "section") {
```

Comment:
Hard undefined like this, @cynthia ?

Build still working after this change https://travis-ci.org/fititnt/html51-translation-testing/builds/174542653#L631